controllers/homework.js

```diff
@@ -810,7 +810,11 @@ router.get('/:assignmentId', (req, res, next) => {
 			const parentId = req.params.assignmentId;
 			const parentType = 'tasks';
 			const filesStorageData = await filesStoragesHelper.filesStorageInit(schoolId, parentId, parentType, req, true);
-			res.render('homework/assignment', { ...assignment, ...renderOptions, ...filesStorageData });
+
+			const numberOfSubmittedSubmissions = submissions.data.filter((submission) => submission.submitted).length;
```

Comment:
Fix es bug in this display
![image](https://user-images.githubusercontent.com/106820326/206278387-89bdf07e-54a1-4cca-a9c7-2f2be0e6e237.png)

Reply:
Leider noch WIP. Wegen Gruppenabgabe.